django_mongodb/compiler.py

```diff
@@ -0,0 +1,291 @@
+from django.conf import settings
+from django.core.exceptions import EmptyResultSet
+from django.db import (
+    DatabaseError,
+    IntegrityError,
+    NotSupportedError,
+    connections,
+)
+from django.db.models import NOT_PROVIDED, Count, Expression
+from django.db.models.constants import LOOKUP_SEP
+from django.db.models.sql import compiler
+from django.db.models.sql.constants import GET_ITERATOR_CHUNK_SIZE, MULTI
+
+from .base import Cursor
+from .query import MongoQuery, safe_call
+
+
+class SQLCompiler(compiler.SQLCompiler):
```

Comment:
I think It would be good to rename this class, I know that some of the methods are public interface so we can't override, execute_sql is one of the thing that We cannot rename, can we?

Reply:
Jib made the same comment shortly after you. Here is my reply in case you missed that thread: These names are names are set by convention (hardcoded) in Django's code, e.g. https://github.com/django/django/blob/85c154da2f07485a1cdc4d886eee4c1a1ef56137/django/db/models/sql/query.py#L226